src/commands/ban.ts

```diff
@@ -24,20 +25,25 @@ export default {
         if (user.id === message.author.id) {
             return message.channel.send("Aww, please don't ban yourself! 💖");
         }
-        /* This checks if the user to be banned
-        is in the guild, and if true,
-        it checks if they can be banned by the bot or not. */
-        if (!message.guild.member(user)?.bannable) {
-            return message.channel.send(
-                '❌ I cannot ban this user! \n**Please make sure that my highest role is above theirs.**',
-            );
-        }
+
+        /* Attempts to ban the user, if the ban is successful,
+        the bot will send a message indicating it was successful, otherwise an error message will be sent
```

Comment:
I think that it would be best to split this line into two, in a way similar to the one above. This comment goes hand-in-hand with the one below for line 31, so please keep that in mind.

```suggestion
        the bot will send a message indicating it was successful, 
        otherwise an error message will be sent */
```